packages/react-dom/src/__tests__/ReactDOMSelect-test.js

```diff
@@ -1289,5 +1289,115 @@ describe('ReactDOMSelect', () => {
           ' This value must be coerced to a string before using it here.',
       );
     });
+
+    it('should not warn about missing onChange if value is not set', () => {
+      ReactTestUtils.renderIntoDocument(
+        <select>
+          <option value="monkey">A monkey!</option>
+          <option value="giraffe">A giraffe!</option>
+          <option value="gorilla">A gorilla!</option>
+        </select>,
+      );
+    });
+
+    it('should not warn about missing onChange if value is undefined', () => {
+      ReactTestUtils.renderIntoDocument(
+        <select value={undefined}>
+          <option value="monkey">A monkey!</option>
+          <option value="giraffe">A giraffe!</option>
+          <option value="gorilla">A gorilla!</option>
+        </select>,
+      );
+    });
+
+    it('should not warn about missing onChange if onChange is set', () => {
+      const change = jest.fn();
+
+      ReactTestUtils.renderIntoDocument(
+        <select value="monkey" onChange={change}>
+          <option value="monkey">A monkey!</option>
+          <option value="giraffe">A giraffe!</option>
+          <option value="gorilla">A gorilla!</option>
+        </select>,
+      );
+    });
+
+    it('should not warn about missing onChange if disabled is true', () => {
+      ReactTestUtils.renderIntoDocument(
+        <select value="monkey" disabled={true}>
+          <option value="monkey">A monkey!</option>
+          <option value="giraffe">A giraffe!</option>
+          <option value="gorilla">A gorilla!</option>
+        </select>,
+      );
+    });
```

Comment:
Why is there no `expect` statement? If these blocks will be executed and print the error, would all of these tests fail?